Index: docker-17.03.1-ce/daemon/daemon_test.go
===================================================================
--- docker-17.03.1-ce.orig/daemon/daemon_test.go
+++ docker-17.03.1-ce/daemon/daemon_test.go
@@ -158,6 +158,7 @@ func TestValidContainerNames(t *testing.
 }
 
 func TestContainerInitDNS(t *testing.T) {
+	t.Skip("not appropriate for a buildroot environment")
 	tmp, err := ioutil.TempDir("", "docker-container-test-")
 	if err != nil {
 		t.Fatal(err)
