Index: docker-17.04.0-ce-rc1/distribution/xfer/transfer_test.go
===================================================================
--- docker-17.04.0-ce-rc1.orig/distribution/xfer/transfer_test.go
+++ docker-17.04.0-ce-rc1/distribution/xfer/transfer_test.go
@@ -9,6 +9,7 @@ import (
 )
 
 func TestTransfer(t *testing.T) {
+    t.Skip("not appropriate for a buildroot environment")
 	makeXferFunc := func(id string) DoFunc {
 		return func(progressChan chan<- progress.Progress, start <-chan struct{}, inactive chan<- struct{}) Transfer {
 			select {
